force-app/components/emailConfirmationModal/classes/EmailConfirmationModalHelperPreview.cls

```diff
@@ -28,10 +28,17 @@ public with sharing class EmailConfirmationModalHelperPreview {
 
 		Map<String, String> replacingMap = new Map<String, String>();
 		for ( String sObjectField : sObjectFields ) {
-			String key = '\\{\\{\\{' + sObjectType + '.' + sObjectField + '\\}\\}\\}';
-			String value = String.valueOf( data[0].get( sObjectField ) );
-
-			replacingMap.put( key, value );
+            if (sObjectField == 'RegistrationFromDateTime__c') {
+                String key = '\\{\\{\\{' + sObjectType + '.' + sObjectField + '\\}\\}\\}';
+                Datetime dateGMT = Datetime.valueOf( data[0].get( sObjectField ) );
+                String value = dateGMT.format( 'dd/MM/yyyy hh:mm' );
+                replacingMap.put( key, value );
+            }
+            else {
+			    String key = '\\{\\{\\{' + sObjectType + '.' + sObjectField + '\\}\\}\\}';
+                String value = String.valueOf( data[0].get( sObjectField ) );
+                replacingMap.put( key, value );
+            }
```

Comment:
```suggestion

                String key = '\\{\\{\\{' + sObjectType + '.' + sObjectField + '\\}\\}\\}';

                if ( isDateField ( sObjectType, sObjectField ) ) {
                    Datetime dateGMT = Datetime.valueOf( data[0].get( sObjectField ) );
                    String value = dateGMT.format( 'dd/MM/yyyy hh:mm' );
                    replacingMap.put( key, value );
                } else {	    
                    String value = String.valueOf( data[0].get( sObjectField ) );
                    replacingMap.put( key, value );
                
```


1. Husk formatering! (jeg skal få fikset en formatter til alle)
2. Gjenbruk kode hvis du ser likheter
3. Det finnes flere dato-felter på kurs, og denne komponenten skal være gjenbrukbar på alle mulige objekter. Dermed må du lage en generell funksjon `isDateField` som sjekker om feltet er et dato-felt.
    - Sjekk både date og datetime
    - Lag tester for funksjonen, slik at du vet den gjør det du forventer